force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -30,354 +30,439 @@
+    results.add(String.valueOf(johnDoe.getAge()));
+    return results; // Replace null with the variable you used to store the result
+  }
+
+  /**
+   * Question 16
+   * Create a Person object with the provided name and age.
+   * @param name The name of the person.
+   * @param age The age of the person.
+   * @return A new instance of the Person class.
+   */
+  public static Person createPerson(String name, Integer age) {
+    // Create a new instance of the Person class using the provided name and age
+    Person newPerson = new Person(name, age);
+    // Return the new instance of the Person class
+    return newPerson; // Replace null with the variable you used to store the result
+  }
+
+  /**
+   * Question 17
+   * Create a method that constructs multiple Person objects.
+   * You are to create a new Person object for each name in the provided List<String> of names,
+   * using the same age for all Person objects. The method should return a List<Person> of all the Person objects created.
+   * @param names A list of names.
+   * @param age The age to be set for all the Person objects.
+   * @return A list of new Person objects.
+   */
+  public static List<Person> createMultiplePersons(
+    List<String> names,
+    Integer age
+  ) {
+    // Loop through each name in the provided list of names
+    List<Person> personLst = new List<Person>();
+    // Create a new Person object for each name
+    for (String name : names) {
+      // Add the new Person object to the list of Person objects
+      personLst.add(new Person(name, age));
     }
 
-
-    /**
-     * Question 18
-     * Compare two Person objects based on their ages.
-     * If the first Person is older, the method should return the first Person. If the second Person is older, it should return the second Person.
-     * If they are the same age, return either.
-     *
-     * @param person1 The first Person object.
-     * @param person2 The second Person object.
-     * @return The Person object of the older person.
-     */
-    public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+    // Return the list of Person objects
+    return personLst;
+  }
+
+  /**
+   * Question 18
+   * Compare two Person objects based on their ages.
+   * If the first Person is older, the method should return the first Person. If the second Person is older, it should return the second Person.
+   * If they are the same age, return either.
+   *
+   * @param person1 The first Person object.
+   * @param person2 The second Person object.
+   * @return The Person object of the older person.
+   */
+  public static Person getOlderPerson(Person person1, Person person2) {
+    if (person1.getAge() > person2.getAge()) {
+      return person1;
+    } else if (person2.getAge() > person1.getAge()) {
+      return person2;
+    } else {
+      return person1;
```

Comment:
The else if check is unnecessary. You can return person1 if person1's age is greater than or equal to person2's, otherwise you can return person2